src/assets/sass/pandora/base/_form.scss

```diff
@@ -223,7 +223,7 @@
     border: 1px solid $offwhite;
     border-radius: $border-radius;
     color: $gray-light;
-    padding: 3rem 4rem;
+    padding: 2rem 2rem;
```

Comment:
Can you rewrite this to: 
`padding: 2rem;`

Reply:
Done